ecommers/lib/core/app_services/product_service.dart

```diff
@@ -9,23 +9,23 @@ class ProductService {
   Future<List<Product>> fetchProducts({
     Categories category,
     String subCategory,
-    int rangeFrom,
-    int rangeTo,
+    int from,
+    int to,
     String searchQuery,
     SortType sortType,
   }) async {
     return productDataRepository.getProducts(
       category: EnumToString.parse(category),
       subCategory: subCategory,
-      rangeFrom: rangeFrom,
-      rangeTo: rangeTo,
+      rangeFrom: from,
```

Comment:
I don't like name 'from', 'to'.  I think 'beginRange' is better name

Reply:
we've decided, that  from / to enough to understand